README.md

```diff
@@ -92,3 +92,35 @@ See
 [comment_block.rs](https://github.com/cobalt-org/liquid-rust/blob/master/crates/lib/src/stdlib/blocks/comment_block.rs)
 for what a block implementation looks like.  You can then register it by
 calling `liquid::ParserBuilder::block`.
+
+### Using partials
+
+To use `{% include %}` or `{% render %}` tags in your templates, you would first need to compile those
+as template partials.
+
+Example:
+
+```liquid
+# common.liquid
+Number: {{ i }}
+```
+
+```rust
```

Comment:
imo this should be a module level comment on `partials`.
- Focuses the documentation in one place
- Verifies the examples

Reply:
Thanks for the quick review! I moved the documentation as a mod rustdoc as you suggested.